mbtiles/src/mbtiles.rs

```diff
@@ -223,7 +303,108 @@ impl Mbtiles {
             self.check_agg_tiles_hashes(&mut conn).await
         }
     }
+    pub async fn statistics<T>(&self, conn: &mut T) -> MbtResult<Statistics>
+    where
+        for<'e> &'e mut T: SqliteExecutor<'e>,
+    {
+        let file_size = query!(
```

Comment:
Seems that `PathBuf` can't work with `file:something?mode=memory` together.  Using sql query for the file size now.